### PR TITLE
Wipe unit tests

### PR DESCRIPTION
**Pull request by rprimet**
Add unit tests for the sensitive info wipe script.

Also refactor slightly some scripts to be shells around library functions

### CODE REVIEW

File: icubam/db/test_wipe.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
+from absl.testing import absltest
+from itertools import repeat
+from sqlalchemy import create_engine
+from icubam.db.fake import populate_store_fake
+from icubam.db.wipe import wipe_db
+from icubam.db.store import Store, StoreFactory
+
+class WipeTest(absltest.TestCase):
+    def setUp(self):
```
Comment (olivierteboul) [inline icubam/db/test_wipe.py:9]: you are using 4 spaces. please stick to 2

File: icubam/db/test_wipe.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      self.engine = create_engine("sqlite:///:memory:", echo=False)
+      store_factory = StoreFactory(self.engine)
+      self.store = store_factory.create()
+      populate_store_fake(self.store)
+
```
Comment (olivierteboul) [inline icubam/db/test_wipe.py:14]: remove blank line

File: icubam/db/test_wipe.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+    def test_wipe(self):
+        with self.engine.connect() as conn:
+            wipe_db(conn.connection, keep_beds=False)
+            users = list(self.store.get_users())
+            self.assertItemsEqual([user.name for user in users],
+                                  repeat("Jean Dumont", len(users)))
```
Comment (olivierteboul) [inline icubam/db/test_wipe.py:21]: i d rather keep the module, i think it s easier to understand when the function comes from: `itertools.repeat(...`

File: scripts/populate_db_fake.py [hunk 4]
```diff
@@ -16,68 +16,7 @@ def main(unused_argv):
...
-
-  add_icu('Beaujon', '93', 'Bobigny', 48.90833333333333, 2.310277777777777,
-          'test_tel', 5, 6, 12, 200, 34, 7, 1)
-  icu_id = add_icu('Bicetre', '93', 'Kremelin-Bicetre', 48.81,
-                   2.353888888888889, 'test_tel', 9, 2, 12, 200, 34, 44, 1)
-  store.add_user_to_icu(
-      admin_user_id, icu_id,
-      User(name='user1', telephone='+336666666', description='desc1'))
-
+  populate_store_fake(store)
```
Comment (olivierteboul) [inline scripts/populate_db_fake.py:19]: good move!

File: scripts/wipe_sensitive_info_db.py [hunk 5]
```diff
@@ -68,7 +35,7 @@ def main(argv):
     ):
       sys.exit(0)
 
-  wipe_db(argv[1])
+  wipe_db_path(argv[1], FLAGS.keep_beds)
```
Comment (olivierteboul) [inline scripts/wipe_sensitive_info_db.py:38]: i would rather use FLAGS for both arguments, i think it s more readable and more consistent

File: scripts/wipe_sensitive_info_db.py [hunk 6]
```diff
@@ -68,7 +35,7 @@ def main(argv):
     ):
       sys.exit(0)
```
Comment (olivierteboul) [inline scripts/wipe_sensitive_info_db.py:36]: no need to sys.exit(0) inside a function, you can just return.

File: icubam/db/wipe.py [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+  -------
+  None
+  """
+  cur = conn.cursor()
+  if not keep_beds:
+    ALTERATIONS["bed_counts"].extend([("n_covid_free", 2), ("n_covid_occ", 4)])
+  for alteration in ALTERATIONS.items():
+    affectations = ",".join([f"{k} = {v}" for (k, v) in alteration[1]])
+    query = f"UPDATE {alteration[0]} SET {affectations};"
+    print(query)
```
Comment (olivierteboul) [inline icubam/db/wipe.py:47]: remove print?

File: scripts/wipe_sensitive_info_db.py (hunk 5 above)
Comment (rprimet) [reply scripts/wipe_sensitive_info_db.py:38]: Hmmm OK but the filename is non-optional, whereas `keep_beds` is optional and therefore a flag?

File: icubam/db/test_wipe.py (hunk 1 above)
Comment (rprimet) [reply icubam/db/test_wipe.py]: Will do, thanks!

File: icubam/db/test_wipe.py (hunk 3 above)
Comment (rprimet) [reply icubam/db/test_wipe.py]: Yup seems clearer that way, thanks

File: scripts/wipe_sensitive_info_db.py (hunk 6 above)
Comment (rprimet) [reply scripts/wipe_sensitive_info_db.py:36]: OK, I thought it'd be more consistent with the `sys.exit(-1) above`

File: scripts/wipe_sensitive_info_db.py (hunk 5 above)
Comment (dulacarnold) [reply scripts/wipe_sensitive_info_db.py:38]: You can make the flag required with absl.  Agreed the flags should generally be kept optional according to argparse/general unix culture, but I also tend to like explicit flags vs. positional arguments.

File: scripts/wipe_sensitive_info_db.py (hunk 6 above)
Comment (dulacarnold) [reply scripts/wipe_sensitive_info_db.py:36]: Ok, if you are leveraging posix return codes this makes sense to me.

File: scripts/wipe_sensitive_info_db.py (hunk 5 above)
Comment (rprimet) [reply scripts/wipe_sensitive_info_db.py]: @dulacarnold @olivierteboul OK, filename is now a required flag

File: scripts/wipe_sensitive_info_db.py (hunk 6 above)
Comment (rprimet) [reply scripts/wipe_sensitive_info_db.py]: The sys.exit(-1) was rendered unnecessary by moving to an all-flags solution so I changed that as well to match your suggestion, thanks!

**Pull request merged**